src/server/system_services/account_server.js

```diff
@@ -807,17 +807,23 @@ async function update_external_connection(req) {
         "sync_credentials_cache.$.access_key": identity,
         "sync_credentials_cache.$.secret_key": encrypted_secret,
     };
+    const map_obj = {
```

Comment:
`map_obj` is not a very meaningful name. rename to something like `ns_resource_updates` or something similar that describes the purpose of the object